frontend/src/styles/components.css

```
@@ -4,10 +4,32 @@
+/* Glassmorphism Utilities */
+.glass-panel {
+  background-color: rgba(var(--bg-panel-rgb), 0.7);
+  backdrop-filter: blur(12px);
+  -webkit-backdrop-filter: blur(12px);
+  border: 1px solid rgba(255, 255, 255, 0.05);
+  box-shadow: var(--shadow-md);
+}
+
+.glass-nav {
+  background-color: rgba(var(--bg-app-rgb), 0.8); /* Falls back if var missing */
+  background-color: rgba(var(--bg-panel-rgb), 0.8);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)